src/contacts.cs

```diff
@@ -399,36 +399,47 @@ interface CNContactRelation : NSCopying, NSSecureCoding, INSCopying, INSSecureCo
 	[EditorBrowsable (EditorBrowsableState.Advanced)]
 	interface CNLabelContactRelationKey {
 
+		[Watch (6,0)]
```

Comment:
those were already available in watchOS (2.0+)

```diff
-CONTACTS_EXTERN NSString * const CNLabelContactRelationFather             API_AVAILABLE(macos(10.11), ios(9.0));
+CONTACTS_EXTERN NSString * const CNLabelContactRelationFather                                                       NS_AVAILABLE(10_11, 9_0);
```

That's identical - just different macros (noise) and there's no mention of watchOS - so that can be confusing. It's often (nothing is ever 100% with Apple) because it's implied - i.e. the header/framework is in watchOS ? so all API that are not decorated are _presumed_ to have been available since the beginning (2.0 for watchOS).

Other ways to double check / reduce confusion is to check if they are already in `Xamarin.WatchOS.dll` (or read the generated code). If they are then the introspection tests will verify that the are present (at runtime).

Reply:
Got it. Removed `[Watch (6,0)]`. 👍